tests/test_wave_01.py

```diff
@@ -130,14 +120,9 @@ def test_update_task_not_found(client):
 
     # Assert
     assert response.status_code == 404
-
-    raise Exception("Complete test with assertion about response body")
-    # *****************************************************************
-    # **Complete test with assertion about response body***************
-    # *****************************************************************
+    assert response_body is None
```

Comment:
Excellent! Something to note, using the get_or_404 is really easy, but it makes it so that there is no response body and the user is left with little information as to what went wrong. It may make more sense in other contexts to manually check to see if the id exists so we can provide information in the response body for the user.